midealocal/devices/db/__init__.py

```diff
@@ -26,6 +38,134 @@ class DeviceAttributes(StrEnum):
 class MideaDBDevice(MideaDevice):
     """Midea DB device."""
 
+    _status: ClassVar[dict[int, str]] = {
+        0: "idle",
+        1: "standby",
+        2: "start",
+        3: "pause",
+        4: "end",
+        5: "fault",
+        6: "delay",
+    }
+
+    _dehydration_speed: ClassVar[dict[int, str]] = {
+        0x00: "0",
+        0x01: "400",
+        0x02: "600",
+        0x03: "800",
+        0x04: "1000",
+        0x05: "1200",
+        0x06: "1400",
+        0x07: "1600",
+        0x08: "1300",
+        0xFF: "default",
+    }
+
+    _water_level: ClassVar[dict[int, str]] = {
+        0x01: "Low",
+        0x02: "Mid",
+        0x03: "High",
+        0x04: "4",
```

Comment:
Should be "translated" to a string: `VeryHigh` ?

Reply:
this is the origina value from lua script.

Reply:
Please share the LUA script